subgraphs/aave-v2/src/mappings/utilFunctions.ts

```diff
@@ -0,0 +1,605 @@
+    const assetDataBorrowStable = incentivesControllerContract.try_assets(Address.fromString(market.sToken)).value.value0;
+    const assetDataBorrowVariable = incentivesControllerContract.try_assets(Address.fromString(market.sToken)).value.value0;
+    // Get the emissions per day for the aToken rewards for deposits
+    rewardEmissions[0] = emissionsPerDay(assetDataSupply);
+    // Get the emissions per second for both the sToken and vToken rewards, average them and get the daily emissions for borrows
+    rewardEmissions[1] = emissionsPerDay((assetDataBorrowStable.plus(assetDataBorrowVariable)).div(BIGINT_TWO));
+  } else {
+    log.info('COULD NOT GET REWARD EMISSIONS FROM INC CONT ON MARKET: ' + market.id, []);
+    // The rewardEmissions array returned is already defaulted to zero vals
+  }
+  return rewardEmissions;
+}
+
+export function getOutputTokenSupply (event: ethereum.Event): void {
+  const outputTokenAddr = event.address;
+  const aTokenInstance = AToken.bind(outputTokenAddr);
+  const tryTokenSupply = aTokenInstance.try_totalSupply();
+  if (!tryTokenSupply.reverted) {
+    log.info('OUTPUT TOKEN NOT REVERTED ' + outputTokenAddr.toHexString() + ' ' + tryTokenSupply.value.toString(), []);
+    const market = Market.load(aTokenInstance.UNDERLYING_ASSET_ADDRESS().toHexString());
+    if (!market) return;
+    market.outputTokenSupply = tryTokenSupply.value;
+    market.save();
+    // This funcion only gets called from mint/burn events which only happen following deposits and withdraws which update the market snapshot
+    // Calling the getMarketDailySnapshot function seems to cause some sort of overflow error, so only the outputTokenSupply field needs to be set
+    const snapId = getMarketDailyId(event, market);
+    const snap = MarketDailySnapshot.load(snapId);
+    if (!snap) return;
+    snap.outputTokenSupply = market.outputTokenSupply;
+    snap.save();
+  } else {
+    log.info('OUTPUT TOKEN SUPPLY CALL REVERTED FOR TOKEN ' + outputTokenAddr.toHexString(), []);
+  }
+}
+
+export function initToken (assetAddr: Address): Token {
+  // The token id is set as the asset address
+  let asset = Token.load(assetAddr.toHex());
+  if (asset === null) {
+    // Create a new Token implementation
+    asset = new Token(assetAddr.toHex());
+    // Instantiate the Token with the IERC20 interface in order to access contract read methods
+    const tokenInstance = IERC20.bind(assetAddr);
+    const tryName = tokenInstance.try_name();
+    if (!tryName.reverted) {
+      asset.name = tryName.value;
+    }
+    const trySymbol = tokenInstance.try_symbol();
+    if (!trySymbol.reverted) {
+      asset.symbol = trySymbol.value;
+    }
+    const tryDecimals = tokenInstance.try_decimals();
+    if (!tryDecimals.reverted) {
+      asset.decimals = tryDecimals.value;
+    }
+    asset.save();
+  }
+  return asset as Token;
+}
+
+export function loadRewardToken (assetAddr: Address, market: Market, type: string): RewardToken {
```

Comment:
nit: no space between function name and left parenthesis.

i.e. change `loadRewardToken (` to `loadRewardToken(`

Reply:
Removed spaces before function name and parenthesis, also added lint rule to not expect this space.

Reply:
Removed spaces before function name and parenthesis, also added lint rule to not expect this space.